revista-latinoamericana-de-recursos-naturales.csl

```diff
@@ -0,0 +1,436 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Revista Latinoamericana de Recursos Naturales</title>
+    <title-short>Rev. Lat. Rec. Nat.</title-short>
+    <id>http://www.zotero.org/styles/revista-latinoamericana-de-recursos-naturales</id>
+    <link href="http://www.zotero.org/styles/revista-latinoamericana-de-recursos-naturales" rel="self"/>
+    <link href="http://www.itson.mx/publicaciones/rlrn/Paginas/guia.aspx" rel="documentation"/>
+    <author>
+      <name>Rafael Cervantes Beyra</name>
+      <email>cervantes@unah.edu.cu</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="engineering"/>
+    <issn>1870-0667</issn>
+    <summary>Style based on Harvard 2011, V1</summary>
+    <updated>2016-12-07T18:42:08+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="anonymous">Anon.</term>
+      <term name="no date">n.d.</term>
+      <term name="retrieved">Available</term>
+      <term name="accessed">Consulted:&#160;</term>
```

Comment:
@rcervantesb, the locale section is meant for translations, not for separating punctuation. Could you remove the `:&#160;` in `<term name="accessed">Consulted:&#160;</term>`? You can just add it as a suffix in `<text term="accessed" text-case="capitalize-first" prefix=" ["/>`, and turn that into `<text term="accessed" text-case="capitalize-first" prefix=" [" suffix=":&#160;"/>`.

It would be even better to use

```xml
        <group prefix=" ["  suffix="]" delimiter=":&#160;">
          <text term="accessed" text-case="capitalize-first" />
          <date variable="accessed">
            <date-part name="month" suffix=" "/>
            <date-part name="day" suffix=", "/>
            <date-part name="year"/>
          </date>
        </group>
```

below.